From 82ac7268f746c31d771e584c1c83f93890b33404 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=B6ren=20Tempel?= <soeren+git@soeren-tempel.net>
Date: Tue, 11 Jul 2023 05:18:00 +0200
Subject: [PATCH] go.env: Don't switch Go toolchain version as directed in
 go.mod

We want users and packages to use the version of Go that is provided
in our package repository. We don't want to download pre-built
toolchains from golang.org.

Also note that prior to Go 1.21, pre-built Go binaries are linked
against glibc and hence do not work on Alpine.
---
 go.env | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/go.env b/go.env
index 6ff2b921d4..a106fb4638 100644
--- a/go.env
+++ b/go.env
@@ -7,6 +7,5 @@
 GOPROXY=https://proxy.golang.org,direct
 GOSUMDB=sum.golang.org
 
-# Automatically download newer toolchains as directed by go.mod files.
-# See https://go.dev/doc/toolchain for details.
-GOTOOLCHAIN=auto
+# Don't attempt to switch to a newer toolchains by default.
+GOTOOLCHAIN=local
